src/test/kotlin/no/skatteetaten/aurora/gobo/graphql/vault/VaultQueryTest.kt

```diff
@@ -0,0 +1,142 @@
+package no.skatteetaten.aurora.gobo.graphql.vault
+
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Import
+import org.springframework.core.io.Resource
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.coEvery
+import no.skatteetaten.aurora.gobo.graphql.GraphQLTestWithDbhAndSkap
+import no.skatteetaten.aurora.gobo.graphql.affiliation.AffiliationQuery
+import no.skatteetaten.aurora.gobo.graphql.graphqlDataWithPrefix
+import no.skatteetaten.aurora.gobo.graphql.graphqlDoesNotContainErrors
+import no.skatteetaten.aurora.gobo.graphql.isFalse
+import no.skatteetaten.aurora.gobo.graphql.isTrue
+import no.skatteetaten.aurora.gobo.graphql.queryGraphQL
+import no.skatteetaten.aurora.gobo.integration.boober.VaultService
+import no.skatteetaten.aurora.gobo.integration.mokey.AffiliationService
+
+@Import(
+    AffiliationQuery::class,
+    VaultDataLoader::class,
+    VaultListDataLoader::class
+)
+class VaultQueryTest : GraphQLTestWithDbhAndSkap() {
+
+    @Value("classpath:graphql/queries/getVaults.graphql")
+    private lateinit var getVaultsQuery: Resource
+
+    @MockkBean
+    private lateinit var affiliationService: AffiliationService
+
+    @MockkBean
+    private lateinit var vaultService: VaultService
+
+    private val vaultSimple = Vault(
+        name = "boober",
+        hasAccess = true,
+        permissions = listOf("APP_PaaS_utv"),
+        secrets = mapOf(Pair("latest.properties", "QVRTX1VTRVJOQU1FPWJtYwp"))
```

Comment:
I stedet for Pair her, kan du bruke formatet:
"latest.properties" to "QVRTX1VTRVJOQU1FPWJtYwp"

Dette vil uansett bli til en Pair til slutt

Reply:
OK